Backbase/variants
# Fix/setup command

### What does this PR do
- Ensures a Xcode project is properly modified when running the command `variants setup`.

### How can it be tested
Among other things:
- Run `variants setup` on a new or existing project that doesn't use Variants yet.
- The project should now contain a `Variants/` folder.
- The project should now use `variants.xcconfig` for its `debug` and `release` configurations.
- The file `project.pbxproj` should have changes pointing default values to values specified in `variants.xcconfig`.
- The project's icon should match the one specified in `variants.yml`.

### Task
resolves #209 

### Checklist:
